parsers/src/main/java/vct/parsers/ColCParser.scala

```diff
@@ -24,7 +24,8 @@ case class ColCParser(override val originProvider: OriginProvider,
                       cc: Path,
                       systemInclude: Path,
                       otherIncludes: Seq[Path],
-                      defines: Map[String, String]) extends Parser(originProvider, blameProvider) with LazyLogging {
+                      defines: Map[String, String],
+                      language: Language) extends Parser(originProvider, blameProvider) with LazyLogging {
```

Comment:
Let's not add language information to transformations. I see that you primarily use it to interpret memory locality based on the language, but rather the AST should speak for itself.